bin/paypal-settlement/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/brave-intl/bat-go/settlement/paypal"
+	"github.com/shopspring/decimal"
+)
+
+var input = flag.String("in", "", "the file or comma delimited list of files that should be utilized")
+var currency = flag.String("currency", "", "a currency must be set")
+var auth = flag.String("auth", "", "an authorization bearer token must be set")
```

Comment:
it's easier to just read this from the environment, like we currently do for uphold API keys